odoo/addons/base/models/ir_fields.py

```diff
@@ -203,8 +210,80 @@ def _str_to_json(self, model, field, value):
             raise self._format_import_error(ValueError, msg, value)
 
     def _str_to_properties(self, model, field, value):
-        msg = _("Unable to import field type '%s'  ", field.type)
-        raise self._format_import_error(ValueError, msg)
+        if isinstance(value, str):  # Get be already convert into list by _extract_records
+            try:
+                value = json.loads(value)
+            except ValueError:
+                msg = _("'%(value)s' does not seem to be a valid JSON for field '%%(field)s'")
+                raise self._format_import_error(ValueError, msg, {'value': value})
+
+        if not isinstance(value, list):
+            msg = _("'%(value)s' does not seem to be a valid Properties value for field '%%(field)s'. A Properties should contain a list of property. Each property need at least 'name', 'type' and 'string' attribute")
+            raise self._format_import_error(ValueError, msg, {'value': value})
+
+        warnings = []
+        for property_value in value:
+            if {'name', 'type', 'string'} > property_value.keys():
+                msg = _("'%(value)s' does not seem to be a valid Properties value for field '%%(field)s'. A Properties should contain a list of property. Each property need at least 'name', 'type' and 'string' attribute")
+                raise self._format_import_error(ValueError, msg, {'value': property_value})
+
+            val = property_value.get('value')
+            if not val:
+                continue
+
+            if property_value['type'] == 'selection':
+                # either label or the technical value
+                new_val = next(iter(
+                    sel_val for sel_val, sel_label in property_value['selection']
+                    if val in (sel_val, sel_label)
```

Comment:
Should you match by value and label? Isn't matching just by value enough?
It would also be simpler:  `if any(val == sel_val for sel_val, _sel_label in property_value['selection'])`.

Reply:
> Should you match by value and label? Isn't matching just by value enough?

Yes, same than for the Selection field. It doesn't cost to much to do it anyway.